Automattic/jetpack
# Social: Add post-publish panel for one-click sharing

This change adds a new panel to our post-publish sidebar, that utilizes the Social share icons we created before.
Also added a modal that has information about the quick sharing feature.

## Proposed changes:

*

### Other information:


## Jetpack product discussion


https://github.com/orgs/Automattic/projects/733/views/2?pane=issue&itemId=38469896
## Does this pull request change what data or activity we track or use?


No.
## Testing instructions:






* Open a post in the editor that is already published
* Run `wp.data.dispatch('core/edit-post').togglePublishSidebar();` in the browser console to trigger the post-publish panel. Check the designs of the panel and the functionality.
* Click on the learn more button, it should open up a modal with information about the feature. The share buttons should also work there.

<img width="280" alt="CleanShot 2023-09-21 at 14 04 39 png" src="https://github.com/Automattic/jetpack/assets/36671565/e9790fed-6c14-4128-81cf-5a05ec68c388">

<img width="923" alt="CleanShot 2023-09-21 at 14 04 47 png" src="https://github.com/Automattic/jetpack/assets/36671565/d33c2fa3-90fa-4353-a220-d5a6c5a2c993">
